public/app/features/alerting/unified/components/StateTag.tsx

```diff
@@ -0,0 +1,35 @@
+import { GrafanaTheme } from '@grafana/data';
+import { useStyles } from '@grafana/ui';
+import { PromAlertingRuleState } from 'app/types/unified-alerting/dto';
+import { css, cx } from 'emotion';
+import React, { FC } from 'react';
+
+type Props = {
+  status: PromAlertingRuleState;
+};
+
+const getStyles = (theme: GrafanaTheme) => ({
```

Comment:
I'd like to move the styles under the component as in all other components.

Reply:
done!